CMakeLists.txt

```diff
@@ -34,24 +34,29 @@ if(NOT OPM_COMMON_ROOT)
     get_filename_component(_leaf_dir_name ${PROJECT_BINARY_DIR} NAME)
     get_filename_component(_parent_full_dir ${PROJECT_BINARY_DIR} DIRECTORY)
     get_filename_component(_parent_dir_name ${_parent_full_dir} NAME)
-    if(_parent_dir_name STREQUAL CMAKE_PROJECT_NAME)
-      #looks like <module-name>/<build-dir> is used
-      get_filename_component(_modules_dir ${_parent_full_dir} DIRECTORY)
-      if(IS_DIRECTORY ${_modules_dir}/opm-common/${_leaf_dir_name})
-        set(opm-common_DIR ${_modules_dir}/opm-common/${_leaf_dir_name})
-      endif()
-    elseif(_leaf_dir_name STREQUAL CMAKE_PROJECT_NAME)
-      # we assume <build-dir>/<module-name>
-      if(IS_DIRECTORY ${_parent_full_dir}/opm-common)
-        set(opm-common_DIR "${_parent_full_dir}/opm-common}")
-      endif()
+    #Try if <module-name>/<build-dir> is used
+    get_filename_component(_modules_dir ${_parent_full_dir} DIRECTORY)
+    if(IS_DIRECTORY ${_modules_dir}/opm-common/${_leaf_dir_name})
+      set(opm-common_DIR ${_modules_dir}/opm-common/${_leaf_dir_name})
+		else()
+			string(REPLACE ${PROJECT_NAME} opm-common _opm_common_leaf ${_leaf_dir_name})
```

Comment:
sorry for nitpick but tabs here, spaces in other repos. please settle on one (idc which).

Reply:
That seems to be an outdated diff. The tabs have already been removed and I forced pushed the changes. Sorry for not mentioning this.